### ISSUE TITLE
All Yescrypt algorithms don't detect all gpus

### ISSUE THREAD

**Issue by Robbylikes** [RainbowMiner/RainbowMiner#2543]
I have an update about all of the yescrypt algorithms I'm not sure what's going on but it's not detecting my 30 series RTX gpus anymore. I thought something was strange when I noticed that yescrypt did not bench. Went to config and entered yescryptR8 in "algorithm" and only the 1080s worked. I'm going to try an older driver version see if that fixes the problem.

**Comment by Robbylikes** [comment]
Just restored my previous drivers and can confirm all yescrypt algorithms do not benchmark. When I run the ccminer all GPUs show up. I'll include pictures with and without "algorithm" filled for yescryptr8.

**Comment by Robbylikes** [comment]
Apologies pictures from my phone don't seem to be compatible with GitHub

**Comment by Robbylikes** [comment]
I can confirm the yescrypt algorithms are not are no longer working on all 3 of my 30 series rigs with the current update and previous.

**Comment by RainbowMiner** [comment]
Yes, all Ccminer have restriction to Pascal and older GPUs. I'll try to provide updated miners.

**Comment by RainbowMiner** [comment]
Ok, the two Yescrypt miners are working again for RTX GPUs. I think, there were early driver problems with together with the RTX + CcminerYescrypt. Seems that these problems are gone with newer Nvidia drivers.

**Comment by Robbylikes** [comment]
I'll give it a go. Thank you for looking into this.

**Comment by Robbylikes** [comment]
It works! Hope this fix will be in next update.
